code/src/components/MovieList.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from 'react';
+
+import { POPULAR_LIST_URL } from '../urls';
+import Movie from './Movie';
+
+const MovieList = () => {
+    const [movies, setMovies] = useState([]);
+
+  useEffect(() => {
+    fetch(POPULAR_LIST_URL)
+        .then((response) => response.json())
+        .then((json) => setMovies(json.results));
+    },[])  
+ 
+    return (
+        <section className="movie-list">
+            {movies.map((movie) => (
+                    <Movie key={movie.id} {...movie}/>
```

Comment:
Cool with spreading props 👍 
Also very clean solution with the component inside the map